bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/OpenAPIUtils.java

```diff
@@ -97,18 +123,21 @@ public static ContentResponse sendOpenAPIRequest(Request request)
             if (clientException.getCause() instanceof HttpResponseException) {
                 if (((HttpResponseException) clientException.getCause()).getResponse()
                         .getStatus() == HttpStatus.UNAUTHORIZED_401) {
-                    logger.warn("OpenAPI request unauthorized. Invalid authorization token.");
+                    LOGGER.warn("OpenAPI request unauthorized. Invalid authorization token.");
                     throw new NanoleafUnauthorizedException("Invalid authorization token");
                 }
             }
             throw new NanoleafException(
```

Comment:
Then rather convert it to the more standard way of passing a cause to an exception:

```
throw new NanoleafException("Failed to send OpenAPI request", clientException);
```
This constructor might not yet exists but should just forward the message and the cause to the parent constructor.